scripts/release_info.py

```diff
@@ -156,7 +153,6 @@ def _git_diff_names_only(left, right='HEAD'):
 
     Returns:
         (list): List of files that are different between the two points.
```

Comment:
Just FYI, typeinfo here should be list(str). (I know this was there before, just noticed it and thought I'd mention it.)

Reply:
done